service/src/main/resources/HTMLTemplate.html

```diff
@@ -1,9 +1,9 @@
 <html lang="en">
 <head>
-    <meta charset="UTF-8"/>
-    <title>CASE</title>
+  <meta charset="UTF-8"/>
+  <title>CASE</title>
 </head>
 <body>
-CASE ID: {{caseId}}
+Case data: {{case_details.id}}
```

Comment:
I would love to have whitespace after `{{` and before `}}`. What about:
```suggestion
Case data: {{ case_details.id }}
```